Specs/Scene/FeatureTableSpec.js

```diff
@@ -161,6 +161,23 @@ describe("Scene/FeatureTable", function () {
     expect(featureTable.setPropertyBySemantic(0, "NAME")).toBe(false);
   });
 
+  it("getPropertyTypedArray returns typed array", function () {
```

Comment:
I think we should also have a test to check that this function returns undefined if the property does not exist.